sys_util/src/eventfd.rs

```diff
@@ -0,0 +1,140 @@
+// Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the THIRD-PARTY file.
+
+use std::fs::File;
+use std::os::unix::io::{AsRawFd, FromRawFd, RawFd};
+use std::{io, mem, result};
+
+use libc::{c_void, dup, eventfd, read, write, EFD_NONBLOCK};
+
+/// A safe wrapper around a Linux eventfd (man 2 eventfd).
+///
+/// An eventfd is useful because it is sendable across processes and can be used for signaling in
+/// and out of the KVM API. They can also be polled like any other file descriptor.
+pub struct EventFd {
```

Comment:
What's the status on this crate `vmm-sys-utils`? I'm asking because we cannot merge the PR as it is now, right? I don't think we want to pull `sys_utils/...` into the `vhost` crate.

Reply:
Yes, that's a problem. This crate depends on the vmm-sys-utils and vm-memory.